### PR TITLE
Documentation for Flyway Hub

### PR DESCRIPTION
**Pull request by MatthewFlatt**
Adds a new section for the documentation pages for Flyway Hub.

### CODE REVIEW

**Review by cjheppell — APPROVED** [review]

File: documentation/hub/automation.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+menu: hub_automation
+subtitle: Automating Pre-flight Checks with Flyway Hub
+redirect_from: /documentation/hub-automation
+---
+
+# Automating Pre-flight Checks
+
+If your Pre-flight checks aren't run before every database deployment, their value diminishes and issues can slip through the cracks. We've made automating Pre-flight checks really easy, especially if you are able to use GitHub actions.
```
Comment (andrew-farries) [inline documentation/hub/automation.md]: Why do we capitalise 'Pre-flight' everywhere? Looks weird to me. I'd go with just 'pre-flight'.

File: documentation/hub/automation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Other CI Tools
+
+If you do not wish to use GitHub Actions to automate your Pre-flight checks, it is possible to use another CI tool.
+
+This <a href="https://github.com/red-gate/flyway-hub-migration-test/blob/main/entrypoint.sh">script</a> can be added as a step in your CI provider, which will run your Pre-flight check and report success or failure. The script requires the following to run:
+
+- **Environment variable**: `FLYWAY_HUB_ACCESS_TOKEN` - An access token for Flyway Hub creating by clicking "Configure automated checks".
+- **Environment variable**: `GITHUB_SHA` - The commit SHA that triggered the workflow. For example, *ffac537e6cbbf934b08745a378932722df287a53*
+- **Script argument**: `Project ID` - The ID of your Flyway Hub project, found in the URL when viewing a project
```
Comment (andrew-farries) [inline documentation/hub/automation.md:32]: We should probably also put this id somewhere else other than just the URL. Asking users to grab it from the URL feels a bit fragile.

File: documentation/hub/checks.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## Build
+
+The Build Pre-flight check will test if your migration scripts can be run against a real, empty database. This has many benefits including:
+
+- Validating syntax of all scripts
+- Checking migration script order
+- Highlighting version number conflicts
+- Ensuring new environments can be built from source control
+
+This simple but effective check is often avoided due to having to provision infrastructure and/or write code to set up and tear down the test. Flyway Hub handles this for you, allowing the check to be run easily with 1 click.
```
Comment (andrew-farries) [inline documentation/hub/checks.md]: 1 -> one

File: documentation/hub/security.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Security Measures
+
+All Flyway Hub resources are stored in the Flyway Hub cloud which is a multitenant service. Here’s what we do to protect your data:
+
+* All user data and running instances are isolated from each other through industry-standard approaches:
+  * Process and filesystem containerisation of running workloads
+  * Network isolation between running workloads
+* Connections to the Flyway Hub API use OpenID Connect authentication and JSON Web Tokens to ensure users only see their own projects
+  * Therefore, users may only see and run their own projects
```
Comment (andrew-farries) [inline documentation/hub/security.md:20]: Do we need to mention these two at all? I'd say this is just a given of using any SaaS product. The ability to only interact with your own stuff isn't really an extra security feature.

**Review by andrew-farries — CHANGES_REQUESTED** [review]

File: documentation/hub/automation.md (hunk 1 above)
Comment (MatthewFlatt) [reply documentation/hub/automation.md]: Done

File: documentation/hub/automation.md (hunk 2 above)
Comment (MatthewFlatt) [reply documentation/hub/automation.md:32]: Can change this later

File: documentation/hub/security.md (hunk 4 above)
Comment (MatthewFlatt) [reply documentation/hub/security.md:20]: Think it's fine for now

**Pull request merged**